src/components/Calculator.js

```diff
@@ -1,43 +1,37 @@
 import React from 'react';
+import calculate from '../logic/calculate';
 import '../style.css';
+import KeyBtn from './Kyes';
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
-  We are not supposed to disable Linters. Linters monitor your code and flag programmatic/stylistic errors. Always try to fix them.

Reply:
Thank you to review my codes. I searched a lot but I can't find any solution for this type of error. can you help me to fix this linter without disabling it? 
thank you in advance.